### ISSUE TITLE
Subfinder hangs when scanning amazonaws.com

### ISSUE THREAD

**Issue by Ocraz** [projectdiscovery/subfinder#208]
## What's the problem (or question)?
Subfinder hangs when scanning amazonaws.com

## How can we reproduce the issue?
1. Install with go get -v github.com/projectdiscovery/subfinder/cmd/subfinder
2. Run: subfinder -d amazonaws.com
3. It hangs at asicloud.s3.amazonaws.com

## What are the running context details?
Install method: go get -v github.com/projectdiscovery/subfinder/cmd/subfinder
Install date: December 29th 2019 - banner says v2
Version go: 1.13.5 linux/amd64
OS: 
- Kali rolling, tried 2 Virtual Box instances
- Manjaro 18.1.5, bare metal
All 3 are up to date.

I get no feedback, when scanning it just hangs.

**Comment by bauthard** [comment]
Hi @Ocraz,

```
root@b0x:~# subfinder -silent -d amazonaws.com > amz.txt
```

```
root@b0x:~# wc amz.txt 
596151 amz.txt
```

Valid issue, looks like subfinder failed to exist in this case, but complete the task as we can see the output results in the file.

**Comment by Ocraz** [comment]
Tried the suggested command exactly, but it doesn't seem to finish either...
Strangely, asicloud.s3.amazonaws.com doesn't seem to exist in the results (tried it with Kali with VirtualBox).

**Comment by Ocraz** [comment]
Subfinder also hangs @ curtis.nflxvideo.net

**Comment by arbazkiraak** [comment]
@ `subfinder -d netflix.com`

**Comment by Ice3man543** [comment]
Hey @arbazkiraak, i'll check this out tomorrow and fix this hopefully!

**Comment by Ice3man543** [comment]
Hey @arbazkiraak @Ocraz the issue should be fixed with PR #213. The google certificate transparency source was acting in a weird way giving duplicate ids, which caused it to be stuck in an infinite loop. Hopefully, that should be fixed now. I tested with netflix.com and it worked.

Closing this issue now. Feel free to reopen if the issue arises again!
